apachesolr_angularjs_search.module

```diff
@@ -1175,10 +1175,23 @@ function apachesolr_angularjs_search_uri_exists($uri) {
   return FALSE;
 }
 
+ /**
+ * Change form ids for multiple delete forms.
+ */
+function apachesolr_angularjs_search_forms($form_id, $args) {
+  $forms = array();
+  if(preg_match('/^apachesolr_angularjs_search_delete_form_\d+/', $form_id)) {
+    $forms[$form_id] = array(
+                   'callback' => 'apachesolr_angularjs_search_delete_form_callback'
```

Comment:
Could you please ensure you have only 2 spaces for indentation in this line and a final comma?